wharehauora-wifi-gateway/wharehauora-wifi-gateway.ino

```diff
@@ -4,53 +4,73 @@
 #include <ESP8266WiFi.h>
 #define AP_NAME "WhareSensor"
 
-const char* my_ssid = "";
-const char* my_pass = "";
-const char* my_server = "m12.cloudmqtt.com";
+const char my_ssid[] = "";
+const char my_pass[] = "";
+const char my_server[] = "m11.cloudmqtt.com";
 
-char* mqtt_username = "";
-char* mqtt_password = "";
+char mqtt_username[32];
+char mqtt_password[32];
 
 #define MY_RF24_CS_PIN 2
 
 #define MY_DEBUG
-#define MY_BAUD_RATE 9600
+
+#define MY_BAUD_RATE 115200
 #define MY_RADIO_NRF24
 #define MY_GATEWAY_ESP8266
 #define MY_GATEWAY_MQTT_CLIENT
-#define MY_MQTT_CLIENT_ID "whare"
+#define MY_MQTT_CLIENT_ID mqtt_username
 #define MY_ESP8266_SSID my_ssid
 #define MY_ESP8266_PASSWORD my_pass
 #define MY_CONTROLLER_URL_ADDRESS my_server
-#define MY_MQTT_PUBLISH_TOPIC_PREFIX "/sensors"
-#define MY_MQTT_SUBSCRIBE_TOPIC_PREFIX "/actuators"
+#define MY_MQTT_PUBLISH_TOPIC_PREFIX "/sensors/wharehauora/5"
```

Comment:
good catch. It's hard coded which home this is for.
that's the next thing i need to work out -- to work out which topic to publish to from the user's input.
Our test homes connected to staging all have the topic hard coded in the gateway at the moment.